src/helpers/HTTPTransport.ts

```diff
@@ -0,0 +1,78 @@
+enum METHODS {
+  GET = 'GET',
+  POST = 'POST',
+  PUT = 'PUT',
+  DELETE = 'DELETE'
+}
+
+type KeyValueString = Record<string, string>;
+
+type Options = {
+  method?: METHODS;
+  headers?: KeyValueString;
+  data?: Document | XMLHttpRequestBodyInit | null;
+  timeout?: number;
+};
+
+class HTTPTransport {
+  get = (url: string, options = {}) => this.request(url, { ...options, method: METHODS.GET });
+
+  post = (url: string, options = {}) => this.request(url, { ...options, method: METHODS.POST });
+
+  put = (url: string, options = {}) => this.request(url, { ...options, method: METHODS.PUT });
+
+  delete = (url: string, options = {}) => this.request(url, { ...options, method: METHODS.DELETE });
+
+  qs(data?: KeyValueString) {
```

Comment:
давай все таки назовем полностью, потому что с первого взгляда не понятно, что такое ```qs```
```suggestion
  queryStringify(data?: KeyValueString) {
```